# HG changeset patch
# Parent  146b6e2aee4c4b8dcc2d8f7791c8e1e5795f1cbb
Fix build warnings in the broadcast's module tests and lightsd.c

diff --git a/core/lightsd.c b/core/lightsd.c
--- a/core/lightsd.c
+++ b/core/lightsd.c
@@ -79,7 +79,7 @@
 static void
 lgtd_signal_event_callback(int signum, short events, void *ctx)
 {
-    int i = (int)ctx;
+    intptr_t i = (intptr_t)ctx;
     assert(i >= 0);
     assert(i < (int)LGTD_ARRAY_SIZE(lgtd_signals));
     assert(i < (int)LGTD_ARRAY_SIZE(lgtd_signal_evs));
diff --git a/tests/lifx/broadcast/test_broadcast_write_callback_getifaddrs_fails.c b/tests/lifx/broadcast/test_broadcast_write_callback_getifaddrs_fails.c
--- a/tests/lifx/broadcast/test_broadcast_write_callback_getifaddrs_fails.c
+++ b/tests/lifx/broadcast/test_broadcast_write_callback_getifaddrs_fails.c
@@ -27,12 +27,6 @@
 
 enum { MOCK_SOCKET_FD = 32 };
 static struct event *MOCK_WRITE_EV = (struct event *)0x7061726973;
-static const struct sockaddr_in LIFX_BROADCAST_ADDR = {
-    .sin_family = AF_INET,
-    .sin_addr = { INADDR_BROADCAST },
-    .sin_port = LGTD_STATIC_HTONS(LGTD_LIFX_PROTOCOL_PORT),
-    .sin_zero = { 0 }
-};
 
 static int event_del_call_count = 0;
 
